dhall/src/Dhall/Parser/Expression.hs

```diff
@@ -146,54 +144,73 @@ parsers embedded = Parsers {..}
       where
         alternative0 = do
             _lambda
+            whitespace
             _openParens
+            whitespace
             a <- label
+            whitespace
             _colon
+            nonemptyWhitespace
             b <- expression
+            whitespace
             _closeParens
+            whitespace
             _arrow
+            whitespace
```

Comment:
I wonder whether we should add aliases `whsp` and `whsp1` to reduce the noise a bit.